src/compass/utils/validate_cslc.py

```diff
@@ -110,12 +110,33 @@ def compare_cslc_products(file_ref, file_sec):
         return
 
     # Compare amplitude of reference and generated CSLC products
-    print('Check mean real part difference between CSLC products is < 1.0e-5')
-    assert np.allclose(slc_ref.real, slc_sec.real,
-                       atol=0.0, rtol=1.0e-5, equal_nan=True)
-    print('Check mean imaginary part difference between CSLC products is < 1.0e-5')
-    assert np.allclose(slc_ref.imag, slc_sec.imag,
-                       atol=0.0, rtol=1.0e-5, equal_nan=True)
+    np.seterr(invalid='ignore')
+    diff_real = np.abs((slc_ref.real - slc_sec.real) / slc_sec.real)
+    diff_imag =  np.abs((slc_ref.imag - slc_sec.imag) / slc_sec.imag)
+
+    # Compute total number of pixels different from nan from ref and sec
+    tot_pixels_ref = np.count_nonzero(~np.isnan(np.abs(slc_ref)))
+    tot_pixels_sec = np.count_nonzero(~np.isnan(np.abs(slc_ref)))
+
+    # Check that total number of pixel is the same
+    assert tot_pixels_ref == tot_pixels_sec
+
```

Comment:
```suggestion
    ma_slc_ref = np.ma.masked_array(slc_ref, mask=~ref_nan)
    ma_slc_sec = np.ma.masked_array(slc_sec, mask=~sec_nan)
    diff_real = np.abs((ma_slc_ref .real - slc_ma_slc_sec sec.real) / ma_slc_sec .real)
    diff_imag =  np.abs((ma_slc_ref .imag - ma_slc_sec .imag) / ma_slc_sec .imag)
```
I think it's safer to use masked arrays instead of ignoring all numpy errors. The Nan-mask has already been computed so might as well use it.